source/tags/PlaceholderSvg/PlaceholderSvg.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+export const PlaceholderSvg = (props) => {
+  const {
+    width,
+    height,
+    text,
+    imgColor,
+    textColor,
+    fontFamily,
+    fontSize,
+    fontWeight
+  } = props;
+
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox={`0 0 ${width} ${height}`}
+      width='100%'
```

Comment:
mixed single and double quotes.  please use double for html attributes and single for js properties.